packages/terra-graphs/src/components/Dev.module.scss

```diff
@@ -0,0 +1,144 @@
+body {
```

Comment:
The name of this file `Dev.module.scss` seems a bit confusing to me. Is it only used for the examples in terra-graph-docs? If so, I wonder if it should reside in terra-graph-docs. I think we should name it what it is instead of what it is used for.

Reply:
Since Changing the name and moving scss file to terra-docs-package, it needs some refactoring. So raised a [new issue](https://github.com/cerner/terra-graphs/issues/121) in order to fix this.